client/my-sites/plans/ecommerce-trial/controller.jsx

```diff
@@ -10,6 +11,14 @@ export function trialExpired( context, next ) {
 	const state = context.store.getState();
 	const selectedSite = getSelectedSite( state );
 
+	context.store.dispatch(
+		recordTracksEvent( 'calypso_plan_trial_expired_page', {
+			site_id: selectedSite.ID,
+			was_ecommerce_trial_site: wasEcommerceTrialSite( state, selectedSite.ID ),
+			was_business_trial_site: wasBusinessTrialSite( state, selectedSite.ID ),
```

Comment:
Could we unify the event props here? I see here we're showing `was_ecommerce_trial_site` and `was_business_trial_site` and others with `trial_type`. Maybe it'll be better for us to choose either one of them, what do you think?